rest/models/summaries.go

```diff
@@ -44,6 +44,9 @@ type Session struct {
 	Open                      float64 `json:"open,omitempty"`
 	PreviousClose             float64 `json:"previous_close,omitempty"`
 	Volume                    float64 `json:"volume,omitempty"`
+	EarlyTradingPrice         float64 `json:"early_trading_price,omitempty"`
```

Comment:
I don't think we ever return these fields from the summaries endpoint, so we shouldn't need them, unless you've seen otherwise